tests/console/orphaned_packages_check.pm

```diff
@@ -67,14 +63,18 @@ sub run {
     # Orphans are also expected in JeOS without SDK module (jeos-firstboot, jeos-license and live-langset-data)
     # Save the orphaned packages list to one log file and upload the log, so QA can use this log to report bug
     # Filter out zypper warning messages and release or skelcd packages
-    my @orphans = split('\n',
-        script_output q[zypper --quiet packages --orphaned | tee -a /tmp/orphaned.log |
-         grep -v "^Warning" | grep -v "\(release-DVD\|release-dvd\|openSUSE-release\|skelcd\)" |
-         awk -F \| 'NR>2 {print $3}'], proceed_on_failure => 1, timeout => 180);
+
+    script_run(
+q[zypper --quiet packages --orphaned | grep -v "^Warning" | grep -v "\(release-DVD\|release-dvd\|openSUSE-release\|skelcd\)" | awk -F '\\| ' 'NR>2 {print $3}' > /tmp/orphaned.log],
+        timeout => 180,
+        die_on_timeout => 0
+    );
+    upload_logs('/tmp/orphaned.log', (log_name => 'orphaned.log'));
+    my @orphans = split(/\n/, path('ulogs/orphaned.log')->slurp);
```

Comment:
Why not just `/tmp/orphaned.log` ?

Reply:
Good question! I believed that we can format the file within the VM at the first! but I was wrong.
the file operation is done in kvm host rather than VM.